Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -222,11 +222,11 @@ public class CsharpCompiler : SinglePassCompiler {
       }
     }
 
-    protected override ConcreteSyntaxTree CreateStaticMain(IClassWriter cw) {
+    protected override ConcreteSyntaxTree CreateStaticMain(IClassWriter cw, ref string argsParameterName) {
```

Comment:
Why `ref`?

Reply:
Removed. I used it few commits ago, but now it's useless.